plugins/babel-plugin-package-import/README.md

```diff
@@ -1,30 +0,0 @@
-# @snowpack/babel-plugin-package-import
```

Comment:
This is a super old package that we really don't care about anymore. Not deleting the package, but just no need to maintain it here anymore